src/main/db/controller/impl/level.ts

```diff
@@ -86,4 +96,13 @@ export class LevelDbController extends EventEmitter implements IDatabaseControll
                 });
         });
     }
+
+    private getDatabaseLocation(): string {
+        try {
+            fs.mkdirSync(this.opts.location, {recursive: true});
+        } catch (e) {
+            warn("creating database directories", e);
```

Comment:
Shouldn't this be an error ?

Reply:
This will throw if database already exists